### DISCUSSION TITLE
Disable right click menu

### DISCUSSION THREAD

**Discussion by tonyjara** [vercel/hyper#7034] [Q&A]
Hi, I'm using neovim with hyper, and it's the only terminal besides the default one where I get good mouse support. However, I would love to have an option to disable the right click because it shows up on top of neovims popup and adds some weird interactions. 
![Screen Shot 2023-02-21 at 19 14 15](https://user-images.githubusercontent.com/45184825/220471510-7230adee-48ab-4792-a3d7-c1a8761f7902.png)
![Screen Shot 2023-02-21 at 19 14 24](https://user-images.githubusercontent.com/45184825/220471515-bbc434e8-1335-4f72-a087-3561254135d5.png)

**Comment by roma-marshall** [comment]
hello @tonyjara, this might help you: [here](https://neovim.io/doc/user/options.html#%27mousemodel%27) or [here](https://www.reddit.com/r/neovim/comments/ra5d8a/how_can_i_disable_only_the_mouse_right_click/)
